data/supertuxkart.appdata.xml

```diff
@@ -5,24 +5,27 @@
   <metadata_license>CC0-1.0</metadata_license>
   <project_license>GPL-3.0+</project_license>
   <name>SuperTuxKart</name>
-  <summary>A racing game</summary>
+  <summary>A 3D arcade racer with a variety of characters, tracks, and modes to play.</summary>
   <description>
     <p>
-      Supertuxkart is a free 3D kart racing game, with a focus on having fun over
-      realism. You can play with up to 4 friends on one PC, racing against each
-      other, or try to beat the computer in single-player mode.
+      Karts. Nitro. Action! SuperTuxKart is a 3D open-source arcade racer with a 
+      variety of characters, tracks, and modes to play. Our aim is to create a 
+      game that is more fun than realistic, and provide an enjoyable experience 
+      for all ages.
     </p>
     <p>
-      See the great lighthouse or drive through the sand and visit the pyramids.
-      Race underground or in space, watching the stars pass by. Or rest under the
-      palm trees on the beach, watching the other karts overtake you. But don't eat
-      the bananas! Watch for bowling balls, plungers, bubble gum and cakes thrown by
-      your opponents.
+      See the great lighthouse or drive through the sand and visit the pyramids. 
+      Race underground or in a spaceship, watching the stars pass by. Or rest 
```

Comment:
You can't watch the stars in STK Enterprise. I think this part should be revisited to talk about STK best tracks rather than lighthouse or Shifting Sands

Reply:
Improvements welcome.  If you come up with a good blurb, I'll happily apply it.